### PR TITLE
Fix border color on attachments

### PR DESCRIPTION
**Pull request by bluwave**
refs: MBL-13863
affects: Parent
release note: none

### CODE REVIEW

**Comment by instructure-cx** [comment]
Scan submitted to Checkmarx

File: Parent/Parent/Conversations/ConversationDetailCell.swift [hunk 1]
```diff
@@ -148,7 +148,7 @@ class ConversationDetailCell: UITableViewCell {
 
             button.pin(inside: self)
 
-            layer.borderColor = tintColor.cgColor
+            layer.borderColor = UIColor.named(.backgroundMedium).cgColor
```
Comment (vanwagonet) [inline Parent/Parent/Conversations/ConversationDetailCell.swift]: I know it's a nitpick, but can you use `borderMedium` instead of `backgroundMedium`?

**Comment by inst-danger** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  This pull request will not generate a release note.
  </td>
    </tr>
  </tbody>
</table>


#### Affected Apps: Parent

[MBL-13863](https://instructure.atlassian.net/browse/MBL-13863)

Coverage | New % | Master % | Delta
-------- | ----- | -------- | -----
**Canvas iOS** | 93.43% | 93.43% | 0%
<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

**Comment by instructure-cx** [comment]
Scan submitted to Checkmarx

**Comment by instructure-cx** [comment]
### Checkmarx scan completed
[Full Scan Details](https://cxmgr.inssec.info/CxWebClient/ViewerMain.aspx?scanid=1019455&projectid=17)
#### Checkmarx Scan Summary
Severity|Count
---|---
High|0
Medium|23
Low|0
Informational|0

#### Violation Summary
Severity|Count
---|---

**Comment by instructure-cx** [comment]
Scan submitted to Checkmarx

**Comment by instructure-cx** [comment]
### Checkmarx scan completed
[Full Scan Details](https://cxmgr.inssec.info/CxWebClient/ViewerMain.aspx?scanid=1019471&projectid=17)
#### Checkmarx Scan Summary
Severity|Count
---|---
High|0
Medium|23
Low|0
Informational|0

#### Violation Summary
Severity|Count
---|---

**Pull request merged**